src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/CallTrait.php

```diff
@@ -18,16 +18,17 @@ trait CallTrait
     /**
      * Adds a method to call after service initialization.
      *
-     * @param string $method    The method name to call
-     * @param array  $arguments An array of arguments to pass to the method call
+     * @param string $method       The method name to call
+     * @param array  $arguments    An array of arguments to pass to the method call
+     * @param bool   $returnsClone Whether the call returns the service instance or not
      *
      * @return $this
      *
      * @throws InvalidArgumentException on empty $method param
      */
-    final public function call($method, array $arguments = [])
+    final public function call($method, array $arguments = []/*, bool $returnsClone = false*/)
```

Comment:
No need for the compat layer: the method is final, we *can* add the argument.

Reply:
Done.